baserom.mm.us.rev1.z64:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+ReferrerUrl=E:\Legend of Zelda, The - Majora's Mask (U) [!].zip
```

Comment:
What's this for?

Reply:
I think it's a remnant from the windows file system from copying something from one location to another. Probably should just be deleted.

Reply:
Yeah, that always happens when you copy something over from Windows to WSL. We could consider gitignoring them, but it's better really to just make sure to check for and delete them whenever you copy stuff over.

Reply:
Looks like this file didn't get removed properly.